zrpc/client_test.go

```diff
@@ -9,14 +9,15 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/zeromicro/go-zero/core/discov"
-	"github.com/zeromicro/go-zero/core/logx"
-	"github.com/zeromicro/go-zero/zrpc/internal/mock"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
 	"google.golang.org/grpc/test/bufconn"
+
```

Comment:
Maybe you should sort the imports, go-zero not separate third-party and local dependencies.

Reply:
er...
My IDE set a configuration that follows my team development specification, therefore it will be automatically formatted.
I will try to change the import group manually.

![image](https://user-images.githubusercontent.com/20939027/228826657-2830a2fe-ce5d-45c4-bbc1-f705d8ad60b2.png)

Reply:
done